k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/values.yaml

```diff
@@ -45,16 +45,16 @@ clusterStorageSets: {}
 externalKafka:
   enabled: false
   endpoints:
-    - tcp://kafka.default.svc.cluster.local:9092
+    - tcp://kafka.cortx.svc.cluster.local:9092
   adminUser: admin
   adminSecretName: kafka_admin_secret
 
 # externalConsul allows configuring an external Consul service
 externalConsul:
   enabled: false
   endpoints:
-    - tcp://consul-server.default.svc.cluster.local:8301
-    - http://consul-server.default.svc.cluster.local:8500
+    - tcp://consul-server.cortx.svc.cluster.local:8301
```

Comment:
Perhaps these should be shortened to `tcp://consul-server:8301` (and so on).  As currently implemented, these are set by  `deploy-cortx-cloud.sh` as command line args to `helm install`.